pyhf/commandline.py

```diff
@@ -0,0 +1,17 @@
+import logging
+logging.basicConfig()
+log = logging.getLogger(__name__)
+
+import click
+import json
+from . import readxml
+
+@click.command()
+@click.option('--entrypoint-xml', required=True, prompt='Top-level XML', help='The top-level XML file for the workspace definition.', type=click.Path(exists=True))
+@click.option('--workspace', required=True, prompt='Workspace directory', help='The location of workspace.', type=click.Path(exists=True))
```

Comment:
I think `workspace` is a bit of a misnomer here. The workspace is the result of parsing the xml and root files via `hist2workspace`. Should we use the same name as the `parse` function, i.e. `--rootdir`

Reply:
maybe top-level directory?